src/pages/nft/class/_classId/index.vue

```diff
@@ -77,7 +76,7 @@
                 :collector-count="ownerCount"
               />
             </template>
-            <template v-else-if="isShowPriceSection">
+            <template v-if="isShowPriceSection">
```

Comment:
I think `<NFTPagePriceSection/>` should now move b4 `<NFTPagePrimitiveClassInfoSection>`